Analysis/Utkscan/CMakeLists.txt

```diff
@@ -8,11 +8,13 @@ option(PAASS_UTKSCAN_GAMMA_GATES "Gamma-Gamma gates in the Gamma-Ray Processors"
 option(PAASS_UTKSCAN_ONLINE "Options for online scans" OFF)
 option(PAASS_UTKSCAN_TREE_DEBUG "Debugging info for TreeCorrelator" OFF)
 option(PAASS_UTKSCAN_VERBOSE "Make Scan More Verbose" OFF)
+option(PAASS_OVERRIDE_EXE_PREFIX "Override the scan exe prefix" OFF)
```

Comment:
These upgrades should be very loud at the end of the compilation (or install) step to alert the user that the default has changed. To avoid a situation where someone sets this up and has other users who might need to recompile and forget or don't know that the executable's name has changed

Reply:
doneish